pkg/logger/logger.go

```diff
@@ -77,14 +77,17 @@ func (l *ZapLogger) Fatal(msg string, fields ...zap.Field) {
 }
 
 func (l *ZapLogger) DebugWithContext(ctx context.Context, msg string, fields ...zap.Field) {
+	fields = append(fields, ctxzap.TagsToFields(ctx)...)
```

Comment:
Do we need to worry about leaking PII and secrets - may test to ensure it does not log anything sensitive.